nservicebus/hosting/cloud-services-host/acs-host-7to8.md

```diff
@@ -0,0 +1,52 @@
+---
+title: NServiceBus Azure Host Upgrade Version 7 to 8
+summary: Instructions on how to upgrade NServiceBus Azure Host Version 7 to 8.
+reviewed: 2017-09-21
+component: CloudServicesHost
+related:
+ - nservicebus/upgrades/6to7
+ - samples/azure/self-host
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 6
+ - 7
+---
+
+The NServiceBus Azure Host will be deprecated as of Version 9 and users are recommended to switch to self hosting for new endpoints. Upgrading existing endpoints is still supported for Version 8
+
+## Incompatible with NuGet 4 and higher
+
+NuGet no longer support packages adding source files and modifying project files. This means that installing the host won't result in a runnable endpoint. Existing endpoints using the host can be upgraded without issues.
```

Comment:
is this only currently true for the new project system?

Reply:
yes, see https://docs.particular.net/nservicebus/upgrades/host-7to8#installation-incompatible-with-nuget